packages/frontend/src/app/heat-map/heat-map.component.spec.ts

```diff
@@ -217,6 +217,10 @@ describe('HeatMapComponent', () => {
   }));
 
   it('should show the tooltip when the mouse hovers a cell', fakeAsync(() => {
+    mockCOMService.fetchBatches = () => of(mockBatches.none);
```

Comment:
Oh good, looks like you fixed those flaky tests! That's such a relief, they were getting pretty annoying.